### DISCUSSION TITLE
Invalid syntax for timestamp type

### DISCUSSION THREAD

**Discussion by gmwill934** [prisma/prisma#12293] [Q&A]
Hey,

Having some trouble inserting a record using `executeRaw` with **postgres**. Whenever I try to execute this I get the following error `db error: ERROR: invalid input syntax for type timestamp with time zone: "$1"`

(the `date` field is of type **`tstzrange`**)

```
const { from, to } = createAppointmentDto;
await prisma.$executeRaw`insert into public."Appointment" ("date", "patientId", "itemId") values ('[${from}, ${to})', ${id}, ${itemId});`;
```

The weird part is that if I hardcode the dates directly as strings, IT WORKS!

```
await prisma.$executeRaw`insert into public."Appointment" ("date", "patientId", "itemId") values ('[2010-01-01 14:45, 2010-01-01 15:45)', ${id}, ${itemId});`;
```

Using `prisma` and `@prisma/client` 3.10.0 

I've also tried to convert the strings to **_dates_** with no luck.

Surely I'm missing something but I really don't know what it is.

Thoughts?

**Comment by austincrim** [comment]
Hey @gmwill934 👋,

What type are `from` and `to` at this point in the app? Are they strings already? Do they look exactly the same as your hardcoded representation?

**Reply by gmwill934** [reply]
Exactly the same.

**Reply by gmwill934** [reply]
I also tried parsing the string as a Date and get the same behavior.

**Reply by austincrim** [reply]
In that case, I would open a new [bug report](https://github.com/prisma/prisma/issues/new?assignees=&labels=kind%2Fbug&template=bug_report.yml) with as much info as possible (possibly including a minimal reproduction) and our engineering teams will take it from there!

**Comment by anaecouto** [comment]
Isn't a ']' missing?

**Reply by tlantli** [reply]
that's how the data type works.

**Reply by janpio** [reply]
I also had to google the Postgres docs to confirm this, but indeed: https://www.postgresql.org/docs/current/rangetypes.html
```
CREATE TABLE reservation (room int, during tsrange);
INSERT INTO reservation VALUES
    (1108, '[2010-01-01 14:30, 2010-01-01 15:30)');
```

**Reply by gmwill934** [reply]
Is there any update on this? 😅

**Reply by gmwill934** [reply]
Or maybe other suggestions?

**Reply by gmwill934** [reply]
I'll try this with the newer versions of prisma

**Comment by ludralph** [comment]
Hi there,

To keep our discussions organized and focused on the most relevant topics, we’re reviewing and tidying up our backlog. As part of this process, we’re closing discussions that haven’t had any recent activity and appear to be outdated.

If this discussion is still important to you or unresolved, we’d love to hear from you! Feel free to reopen it or start a new one with updated details.

For more details about our priorities and vision for the future of Prisma ORM, check out our latest blog post: https://www.prisma.io/blog/prisma-orm-manifesto.

Thank you for your understanding and being part of the community!
